src/locales/de.ts

```diff
@@ -1379,8 +1379,7 @@ export default {
 		"Das Thema wurde erfolgreich kopiert.",
 	"pages.room.copy.task.message.copied":
 		"Die Aufgabe wurde erfolgreich kopiert.",
-	"pages.room.itemDelete.text":
-		'Bist du dir sicher, dass du das Element "{itemTitle}" löschen möchtest?',
+	"pages.room.itemDelete.text": "{itemTitle} {itemType} wirklich löschen?",
```

Comment:
```suggestion
	"pages.room.itemDelete.text": "{itemType} {itemTitle} wirklich löschen?",
```
I guess `itemType` and `itemTitle` need to switch places here in order to be correct.